File: PBO1.cpp
#include<iostream>
using namespace std;

int main (){								//main program
	char g[6]={'Q','W','E','R','T', 'Y'};
	int f[3][2] = {{9,8}, {7,6}, {5,4}};
	int a,j=1,m,d;									//tipe data yang mendeklarasikan variabel
	char angka;
	
	cout<<endl;
	cout<<"Array 1D & Perulangan For"<<endl;
    for(int a=0; a<6; a++){									//Perulangan For
        cout<<"Isi Array ke-"<<a<<" = "<<g[a]<<endl;             //output
    }
    
    cout<<endl;
    cout<<"Array Multidimensi"<<endl;		//Array Multidimensi (2 dimensi)
    for(int j=0;j<3;j++)
	{
		for(int k=0;k<2;k++)
		{
			cout<<f[j][k]<<"\t";
		}
		cout<<endl;
	}
    	cout<<endl;
    	cout<<"Percabangan IF menentukan angka Ganjil/Genap"<<endl;       //Percabangan IF
    	cout<<"Masukkan Angka (1-10): ";    //Inputan
    	cin>>angka;
    if (angka%2==0){
        cout<<angka<<" Adalah angka genap"<<endl;   //Output
    }else{
        cout<<angka<<" Adalah angka ganjil"<<endl;  //Output
	}
	
	cout<<endl;
	cout<<"Perulangan While Do While"<<endl;			//Do While
	int b;
	while (b < 3) {
        cout << "Konversi Satuan jam ke Menit"<<endl;
        b++;
    }
	do {
		m= j*60;
    	d=m*60;
	 	cout<<j<<" Jam "<<m<<" Menit "<<d<<" Detik"<<endl;
	 	j +=1;	
    } while (j <=3);
	 return 0;
}

File: PBO1.java
import java.util.Scanner;
class Sdatatugas1 {
    public static void main(String args[]) {
        char[] a = {'Q','W','E', 'R','T','Y'};
        int[][] x = {{0,1}, {2,3}, {4,5}};
        int jam = 1;

        Scanner input = new Scanner(System.in);
        System.out.println("ARRAY 1D");
        for(int i=0; i<5; i++){        //perulangan For & array 1D
            System.out.println("Index " + i + " = " + a[i]);
        }

        System.out.println("ARRAY 2D");
        for(int i=0;i<3;i++)
        {
            for(int j=0;j<2;j++)
            {
                System.out.print(x[i][j] + "\t");
            }
            System.out.println();
        }
        System.out.println("Percabangan IF menentukan angka Ganjil/Genap");
        System.out.print("Nilai : ");
        int n = input.nextInt();                //Input
        System.out.println("Nilainya : " + n);  //output

        if (n % 2 == 0) {
            System.out.println("Bilangan Genap");
        } else {
            System.out.println("Bilangan Ganjil");
        }

        int f = 0;
        while (f < 3) { //While
            System.out.println("Konversi jam ke menit");
            f++;
        }
        int e = 0;
        do {            //Do while
            int m = jam * 60;
            int d = m * 60;
            System.out.println(jam + " Jam " + m + " Menit " + d + " Detik");
            jam++;
            e++;
        }
        while (e < 3);
    }
}